src/main/java/com/minecolonies/coremod/configuration/ConfigurationHandler.java

```diff
@@ -38,6 +38,7 @@ public static synchronized void init(final File file)
         try
         {
             config.load();
+            builderPlaceConstructionTape = config.get(CATEGORY_GAMEPLAY, "placeConstructionTape", builderPlaceConstructionTape, "Should builder place construction tape").getBoolean();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)